src/pytest_workflow/plugin.py

```diff
@@ -117,18 +147,31 @@ def run_workflow(self):
         # Create a workflow and make sure it runs in the tempdir
         workflow = Workflow(self.workflow_test.command, tempdir)
 
-        print("run '{name}' with command '{command}' in '{dir}'".format(
+        # Add an extra print. As it looks better in the pytest output.
+        print()
+        print("'{name}' with command '{command}' in '{dir}' is queued.".format(
```

Comment:
You can also start the second print statement with a newline. That looks a bit cleaner, i think. 
I.e. `print("\n'{name}' with command ......)`